src/transformers/pipelines/zero_shot_image_classification.py

```diff
@@ -120,7 +122,8 @@ def preprocess(self, image, candidate_labels=None, hypothesis_template="This is
         inputs = self.image_processor(images=[image], return_tensors=self.framework)
         inputs["candidate_labels"] = candidate_labels
         sequences = [hypothesis_template.format(x) for x in candidate_labels]
-        text_inputs = self.tokenizer(sequences, return_tensors=self.framework, padding=True)
+        padding = "max_length" if self.model.config.model_type == "siglip" else True
```

Comment:
should be set by default in the tokenizer no? instead of forcing it

Reply:
Which attribute can I use for that? I thought that this is only possible in the call of the tokenizer

Reply:
As discussed offline, the pipeline sets it to True so we can't really do anything but this!